Update app service params after provisioning

santaduda opened the pull request:
After app services are provisioned the app service parameters and KeyVault values are set.

rdevalap commented on create-env.py:
```diff
@@ -50,7 +50,7 @@ def show_help():
         resource_client = ResourceManagementClient(credential, subscription_id)
         web_client = WebSiteManagementClient(
-            credential=DefaultAzureCredential(),
+            credential=DefaultAzureCredential(exclude_shared_token_cache_credential=True),
```
I do not have this issue. Can you rollback this? Keep it DefaultAzureCredential?

rdevalap replied:
Also, test it from the github workflow?

rdevalap replied:
Also, remove the "After" in Keyvault.substitute?

rdevalap reviewed (changes requested):
Hi @santaduda , As discussed,
1. Can you add the parameters for all app services?
2. Can you comment the line `KeyVault.substitue_expression(microservices_config)` in `diagnostic-settings.py`

santaduda commented:
1. Added all microservices and application parameters
2. Removed exclude_shared_token_cache_credential=True in DefaultAzureCredential()
3. Commented line : KeyVault.substitue_expression(microservices_config) in diagnostic-settings.py

rdevalap reviewed (approved):
Tested the code. Works well. 
Provisions app services, configures health check, diagnostic settings enabled
